assets/js/app.js

```diff
@@ -25,7 +25,9 @@ Hooks.CommandInput = {
     const input = document.getElementById('commandInput');
     const sendCursorPosition = e => {
       if (input.selectionStart === input.selectionEnd) {
-        this.pushEventTo('#commandInput', 'caret-position', { position: input.selectionEnd });
+        if (e.keyCode !== 9) {
```

Comment:
Nit: I bet `9` is the keyCode for `tab`. Not critical at this point, but we would want to create a constant or something if this piece of code grows.